client/src/containers/VerifyEmailPage.tsx

```diff
@@ -72,16 +79,28 @@ const VerifyEmailPage = withI18n()((props: withI18nProps) => {
         <Trans render="h4">Check your email inbox & spam</Trans>
         <br />
         <Trans>
-          Click the link we sent to verify your email address {!!user && user.email}. It may take a
-          few minutes to arrive. Once your email has been verified, you’ll be signed up for Data
-          Updates.
+          Click the link we sent to verify your email address. It may take a few minutes to arrive.
```

Comment:
removing the user email here since user could be in expired verification link flow while not logged in